lib/mixlib/install/script_generator.rb

```diff
@@ -171,8 +172,24 @@ def shell_code_from_file(vars)
           %w{.. .. .. support},
           "install_command"
         )
-        Util.shell_code_from_file(vars, fn, powershell,
-                                  http_proxy: http_proxy, https_proxy: https_proxy)
+        code = Util.shell_code_from_file(
+          vars, fn, powershell,
+          http_proxy: http_proxy, https_proxy: https_proxy
+        )
+        prefix = powershell ? powershell_prefix : ""
+        prefix.concat(code)
+      end
+
+      # Prefixes the PowerShell install script with helpers and shell vars
+      # to detect the platform version and architecture.
+      #
+      # @return [String] PowerShell helpers and shell vars for platform info
+      def powershell_prefix
+        [
+          Mixlib::Install::Generator::PowerShell.get_script("helpers.ps1", {}),
```

Comment:
Pretty sure I tried it before without the arg and it complained but I'll double check to verify.